client/platform/web-girder/views/ViewerLoader.vue

```diff
@@ -86,6 +88,7 @@ export default defineComponent({
     ref="viewerRef"
   >
     <template #title>
+      <ViewerAlert />
```

Comment:
I think this might need to move down after the `v-tabs` so that the tab's horizontal position doesn't jump when navigating between data browser and viewer.  What do you think?

Reply:
I personally prefer it before before the tabs for a couple of reasons:
1. I think it gets lost a bit when its at the end of the tabs
2. The placement at the end makes me think the warning is associated with the image/video being viewed.

I am fine with whatever everyone thinks is best though

![Screenshot from 2021-09-22 12-10-16](https://user-images.githubusercontent.com/40494088/134381899-47dcc009-aabe-4007-9c0a-7f3f9b33dfa1.png)
![Screenshot from 2021-09-22 12-09-46](https://user-images.githubusercontent.com/40494088/134381904-b8dfc33d-4135-4765-b3dc-9e2d956af7f2.png)
.